AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -129,8 +132,17 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId flags:(MSFlags)fl
 
 - (NSUInteger)countLogsBeforeDate:(NSDate *)date {
   long long timestampMs = (long long)([date timeIntervalSince1970] * 1000);
-  NSMutableString *condition = [NSMutableString stringWithFormat:@"\"%@\" <= '%lld'", kMSTimestampColumnName, timestampMs];
-  return [self countEntriesForTable:kMSLogTableName condition:condition];
+  NSMutableString *condition = [NSMutableString stringWithFormat:@"\"%@\" <= ?", kMSTimestampColumnName];
+  return [self countEntriesForTable:kMSLogTableName condition:condition withValues:@[ @(timestampMs) ]];
+}
+
+- (NSString *)buildKeyFormatWithCount:(unsigned long)count {
+  NSString *keyFormat = @"(";
+  for (uint i = 0; i < count; i++) {
```

Comment:
`unsigned long`, `uint`, `NSUInteger`. Choose one

Reply:
@annakocheshkova not resolved

Reply:
@MatkovIvan where?

Reply:
![Screen Shot 2019-12-16 at 15 49 36](https://user-images.githubusercontent.com/1836384/70908244-c5070980-201b-11ea-8b7a-9b323d0b38f9.png)